include/constants.h

```diff
@@ -616,11 +616,67 @@
 #define SPEC_MACRO_GLOBAL      "%global"
 
 /**
- * @def SPEC_SECTION_CHANGELOG
+ * @def SPEC_MACRO_AUTOPATCH
  *
- * How the change log is specified in the spec file.
+ * The %autopatch macro used to apply all defined Patches.
  */
-#define SPEC_SECTION_CHANGELOG "%changelog"
+#define SPEC_MACRO_AUTOPATCH   "%autopatch"
+
+/**
+ * @def SPEC_MACRO_AUTOSETUP
+ *
+ * The %autosetup macro used to unpack all Sources and apply all
```

Comment:
Small nit: Sources and Patches capitalized; not sure if we're referencing a "proper" thing or if it's just a rogue shift key here and there.

Reply:
The naming is deliberately.  I was trying to make the comments allude to what someone would see in a spec file.  Like a PatchN: or SourceN: header.